proposed/2023/PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,46 @@
+# Markdown Readme Rendering
+* Start Date: 6/27/23
+* Authors: Allie Barry ([albarry4](https://github.com/albarry4))
+* Issue: https://github.com/NuGet/Home/issues/12583
+
+## Summary
+
+Support rendering markdown readmes in the package details page for NuGet in Visual Studio to make package affordances more accesible for customers browsing packages within the PMUI, and to increase motivation for high quality Readme inclusion for package authors. This will be part of a larger effort to enhance the experience around Readmes and increase their inclusion rate across packages in the NuGet ecosystem. 
+
+## Motivation
+
+In the latest NuGet in Visual Studio 17.5 customer survey, our team found that one of the largest problems that our customers face while using NuGet in the Visual Studio ecosystem is around understanding package affordances – specifically calling out the usage of package metadata such as READMEs. Only 35.35% of survey respondents feel that they can adequately understand the affordances of a package (does it do what I need to to do when I need it?) while browsing and installing packages. Including a detailed README file and implementing features to facilitate README viewing and inclusion would positively impact this number.   
+
+## Explanation
+
+### Functional Explanation
+
+The goal of this new experience would be to integrate the experience of viewing a rendered Readme directly into NuGet in Visual Studio. Additionally, we want to draw more attention to the README in general, and make it feel like an important and crucial element of a package.  
```

Comment:
Should the README be shown by default when viewing a package? In other words, the README tab will be shown first, pushing the package details to a separate tab (and requiring a separate click). 

Seems to work for nuget.org, so I'd guess we're happy with that. It's a performance consideration for the VS side, so that's why I want to be clear on our goal.

Reply:
Part of my goal with this proposal was to contribute to the general push for README inclusion and README awareness within NuGet. I feel that making the README shown by default would both 1) motivate more package authors to include a high-quality readme that demonstrates package affordances and usage information, and in turn 2) make this information more accessible and prevalent for package consumers (I included some findings from the recent survey in the "Motivation" section above that explains why I think this is important). I'm open to hearing any other ideas for a potential layout here as well!

Reply:
Have we asked customers to see what they want to see most prominently? If there are more customers that want to see dependencies & project URL & license, then always showing the readme will provide those customers a worse experience, even if improving readme awareness is a good goal.

Reply:
> Have we asked customers to see what they want to see most prominently? If there are more customers that want to see dependencies & project URL & license, then always showing the readme will provide those customers a worse experience, even if improving readme awareness is a good goal.

Maybe the default tab should be configurable by customers? By default: Readme tab. Could be in VS Options, or a pin on the tabs, or whatever UX likes for this concept.

Reply:
@zivkan not explicitly, but during our interviews on package quality, package documentation (including READMEs and other sources of getting started/bugs/use case information) was mentioned by all interviewees, and was most commonly indicated as the single most important factor (outside of package affordances and legality/security concerns) in determining package quality, and in turn determining which packages to install. This, coupled with the results of the VS 17.5 survey, seemed to me to be sound evidence that customers would like to see README information prominently displayed when viewing package details.